.travis.yml

```diff
@@ -1,3 +1,5 @@
+if: branch = master
```

Comment:
does it include the tags tho?

Reply:
I would think so - these would be `branch=master, tag=true` (or similar, conceptually), wouldn't they?

Reply:
https://docs.travis-ci.com/user/conditions-v1

seems to list `tag` separately. Although we could postpose it until the next release :D

Reply:
AFAICT this should work as I intended - maybe you're aiming for something else?
```
any commit to master (including merge of a PR) => build
any tag on master => build and publish
```

I'd like any commit to master to still trigger a Travis build so that we find out about Travis-specific build issues a bit earlier than release time.

Reply:
Ok, sounds good then :)